AdminWebsite/AdminWebsite.AcceptanceTests/Steps/BookingDetailsSteps.cs

```diff
@@ -98,10 +99,46 @@ private void VerifyJudgeInParticipantsList()
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.JudgeName).Text.Should().Contain(judge.DisplayName);
 
             if (!OnlyDisplayEmailAndUsernameIfCurrentUserMadeTheBooking()) return;
-            _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.ParticipantEmail(hearingJudge.Id)).Text.Should().Be(judge.AlternativeEmail);
+            if (GetJudgeEmail(hearing) != null)
+            {
+                _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.ParticipantEmail(hearingJudge.Id)).Text.Should().Be(judge.AlternativeEmail);
+            }
+            else
+            {
+                _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.ParticipantEmail(hearingJudge.Id)).Text.Should().Be("TBC");
+            }
             _browsers[_c.CurrentUser].Driver.WaitUntilVisible(BookingDetailsPage.ParticipantUsername(hearingJudge.Id)).Text.Should().Be(judge.Username);
         }
 
+        private string GetJudgeEmail(HearingDetailsResponse hearing)
+        {
+            var email = GetOtherInformationObject(hearing.OtherInformation).JudgeEmail;
+            if (email == string.Empty)
+            {
+                return null;
+            }
+            return email;
+        }
+        
+        private static OtherInformationDetails GetOtherInformationObject(string otherInformation)
+        {
+            try
+            {
+                var properties = otherInformation.Split("|");
```

Comment:
tbc it adds empty || for null values?

Reply:
Null or blank values have an empty string yes

Reply:
super